browser/css/jssidebar.css

```diff
@@ -433,3 +433,7 @@ span.jsdialog.sidebar.ui-treeview-notexpandable {
 #TemplatePanel > div:nth-child(5) {
 	height: inherit;
 }
+
+#highlightstyles input {
```

Comment:
please avoid using simply the ID of widget (it can be repeated somewhere)
best to put also parent dialog/sidebar: eg. `#TemplatePanel #highlightstyles input`